### PR TITLE
add ability to create and fetch visualizations

### PR DESCRIPTION
**Pull request by bbrewer-papa-x**
https://github.com/SmartColumbusOS/discovery/issues/8

co-authored-by: Jake Prem <jakeprem@gmail.com>

### CODE REVIEW

**Comment by mcsearchin** [comment]
@cameronmarsh and I (@abrock-accenture ) are carrying this PR forward

File: test/unit/discovery_api_web/controllers/visualization_controller_test.exs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  @valid_jwt_subject AuthHelper.valid_jwt_sub()
+  @user_info_body Jason.encode!(%{"email" => "x@y.z"})
+
+  setup do
+    jwks = AuthHelper.valid_jwks()
+    CachedJWKS.set(jwks)
+
+    bypass = Bypass.open()
+
+    really_far_in_the_future = 3_000_000_000_000
```
Comment (rajones-hntb) [inline test/unit/discovery_api_web/controllers/visualization_controller_test.exs]: great name

File: lib/discovery_api/schemas/generators.ex [hunk 2]
```diff
@@ -0,0 +1,10 @@
+defmodule DiscoveryApi.Schemas.Generators do
+  @moduledoc false
+
+  @size 8
+  @alphabet "abcdefghijklmnopqrstuvwxyz0123456789"
```
Comment (NathanielScottStevens) [inline lib/discovery_api/schemas/generators.ex]: I was noodling to try to find a more clever way of writing this such as `Enum.to_list(?a..?z) ++ Enum.to_list(?1..?9)` but I think they'll all end up harder to read.

File: lib/discovery_api/schemas/visualizations.ex [hunk 3]
```diff
@@ -0,0 +1,24 @@
+defmodule DiscoveryApi.Schemas.Visualizations do
+  @moduledoc """
+  Interface for reading and writing the Visualization schema.
```
Comment (NathanielScottStevens) [inline lib/discovery_api/schemas/visualizations.ex]: When I see the word interface in Elixir I tend to think Behaviour but that may just be me.

**Pull request merged**